stock_picking_whole_scrap/tests/test_stock_picking_whole_scrap.py

```diff
@@ -0,0 +1,83 @@
+# Copyright 2018 Tecnativa - Sergio Teruel
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo.tests.common import SavepointCase, tagged
+from odoo.exceptions import UserError
+
+
+@tagged('post_install', '-at_install')
+class TestStockPickingScrapQuick(SavepointCase):
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.warehouse = cls.env.ref('stock.warehouse0')
+        cls.picking_type_out = cls.env.ref('stock.picking_type_out')
+        cls.customer_location = cls.env.ref('stock.stock_location_customers')
+        cls.partner = cls.env['res.partner'].create({
+            'name': 'Partner - test',
+            'customer': True,
+        })
+        cls.product = cls.env['product.product'].create({
+            'name': 'test',
+            'type': 'consu',
+        })
+        cls.quant = cls.env['stock.production.lot'].create({
```

Comment:
```suggestion
        cls.quant = cls.env['stock.quant'].create({
```
but it seems it's not being used at all or as you force `qty_done`, it's not affected.